src/main/java/chess/domain/Board.java

```diff
@@ -0,0 +1,105 @@
+package chess.domain;
+
+import chess.domain.piece.BishopPiece;
+import chess.domain.piece.Color;
+import chess.domain.piece.EmptyPiece;
+import chess.domain.piece.KingPiece;
+import chess.domain.piece.KnightPiece;
+import chess.domain.piece.PawnPiece;
+import chess.domain.piece.Piece;
+import chess.domain.piece.QueenPiece;
+import chess.domain.piece.RookPiece;
+import chess.domain.position.File;
+import chess.domain.position.Position;
+import chess.domain.position.Rank;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Board {
+
+    private static final int INITIAL_CAPACITY = 64;
+
+    private final Map<Position, Piece> board;
+
+    private Board(final Map<Position, Piece> board) {
+        this.board = board;
+    }
+
+    public static Board create() {
+        final Map<Position, Piece> board = new LinkedHashMap<>(INITIAL_CAPACITY);
+
+        initialPositions(board);
+        initialPieces(board);
+
+        return new Board(board);
+    }
+
+    private static void initialPositions(Map<Position, Piece> board) {
+        List<Position> positions = Stream.of(Rank.values())
+                .flatMap(rank ->
+                        Stream.of(File.values()).map(file -> new Position(file, rank)))
+                .collect(Collectors.toList());
+
+        for (Position position : positions) {
+            board.put(position, new EmptyPiece());
+        }
+    }
+
+    private static void initialPieces(Map<Position, Piece> board) {
+        initialPiecesWithoutPawn(board, Rank.EIGHT, Color.BLACK);
+        initialPawns(board, Rank.SEVEN, Color.BLACK);
+
+        initialPiecesWithoutPawn(board, Rank.ONE, Color.WHITE);
+        initialPawns(board, Rank.TWO, Color.WHITE);
+    }
+
+    private static void initialPiecesWithoutPawn(Map<Position, Piece> board, final Rank rank,
+                                                 final Color color) {
+        board.put(new Position(File.A, rank), new RookPiece(color));
+        board.put(new Position(File.B, rank), new KnightPiece(color));
+        board.put(new Position(File.C, rank), new BishopPiece(color));
+        board.put(new Position(File.D, rank), new QueenPiece(color));
+        board.put(new Position(File.E, rank), new KingPiece(color));
+        board.put(new Position(File.F, rank), new BishopPiece(color));
+        board.put(new Position(File.G, rank), new KnightPiece(color));
+        board.put(new Position(File.H, rank), new RookPiece(color));
+    }
+
+    private static void initialPawns(Map<Position, Piece> board, final Rank rank,
```

Comment:
```suggestion
    private void initialPawns(final Rank rank, final Color color) {
```
팩터리 메서드가 필요한가요?
팩터리 메서드 때문에 static을 사용하면서 이런 메서드 형태가 나온 것 같은데, 생성자를 잘 활용해봅시다.

Reply:
이후 테스트에서 Board에 원하는 체스판 Map을 넣어줄 수 있도록 초기화에 대한 로직을 BoardInitializer 클래스로 분리해봤어요!!
혹시 이러한 부분에 대해서는 어떻게 생각하시나요..?

Reply:
1. 일반적으로 많이 사용하는 팩터리 클래스를 만드는게 낫지 않을까요?
2. Board 객체 생성은 Map만 넘겨주면 개발자가 의도하지 않은 형태의 보드도 만들 수 있겠군요? 외부의 클래스에서 만들어준 맵을 생성자 인자로 받아야 하는 이유를 모르겠네요.